relevanceai/operations_new/vectorize/base.py

```diff
@@ -41,7 +42,7 @@ def transform(self, documents: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
 
         """
 
-        updated_documents = documents
+        updated_documents = deepcopy(documents)
```

Comment:
Why do we need to `deepcopy`?

Reply:
because otherwise docs are editted in place